package.json

```diff
@@ -126,6 +126,7 @@
     "places.js": "1.17.1",
     "prettier": "^2.4.1",
     "prop-types": "15.6.2",
+    "punycode": "^1.4.1",
```

Comment:
what's this needed for?

Reply:
this is added by parcel while adding `@algolia/client-composition`, I'll clean this up when removing it